analysers/analyser_osmosis_highway_vs_building.py

```diff
@@ -128,6 +128,34 @@
 CREATE INDEX idx_{0}water_linestring ON {0}water USING gist(linestring)
 """
 
+sql08 = """
+CREATE TEMP TABLE commercial AS
+SELECT
+    id,
+    geom,
+    CASE
+      WHEN tags?'layer' THEN tags->'layer'
+      ELSE '0'
+    END AS layer,
```

Comment:
Better use `COALESCE` here.

Reply:
Thanks! Implemented!
I used the uppercase `COALESCE` as per your suggestion, but it's used very mixed in uppercase (11x) /lowercase (12x) in all files